### PR TITLE
Bc 2192 class manage text changes for brb and nbc

### PR DESCRIPTION
**Pull request by odalys-dataport**
# Description

BRB changes
https://ticketsystem.dbildungscloud.de/browse/BC-2192

Includes NBC changes:
https://ticketsystem.dbildungscloud.de/browse/BC-2258

## Approval for review

- [ ] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

File: views/administration/forms/form-classes-manage.hbs [hunk 1]
```diff
@@ -56,38 +56,12 @@
     <div class="mb-3">
         <h2 class="h4">{{$t "administration.classes.headline.questionStudentsNotInSystem" (dict "shortTitle" theme.short_title)}}</h2>
         <p>
-           {{$t "administration.classes.text.inviteTheParentsViaLink" }}
+           {{& $t "administration.classes.text.noConsentNecessary" }}
+        </p>
+        <p>
+           {{& $t "administration.classes.text.inviteViaLink" }}
```
Comment (MartinSchuhmacher) [inline views/administration/forms/form-classes-manage.hbs:62]: was it discussed with product team that the other options got removed?

File: helpers/theme.js [hunk 2]
```diff
@@ -33,12 +35,13 @@ const setTheme = (res) => {
 	const documents = defaultDocuments();
 	const baseDir = (res.locals.currentSchoolData || {}).documentBaseDir || documents.documentBaseDir;
 	const themeTitle = instanceSpecificTitle();
+	const shortThemeTitle = instanceSpecificTitle(true);
```
Comment (MartinSchuhmacher) [inline helpers/theme.js:38]: where is the advantage of this whole implementation? the value for `Schul-Cloud Brandenburg` for example can be directly taken from the var `SC_TITLE`

File: helpers/theme.js [hunk 3]
```diff
@@ -7,12 +7,14 @@ const {
 	ALERT_STATUS_URL,
 } = require('../config/global');
 
-const instanceSpecificTitle = () => {
+const instanceSpecificTitle = (short = false) => {
```
Comment (MartinSchuhmacher) [inline helpers/theme.js:10]: see question below

**Review by MartinSchuhmacher — CHANGES_REQUESTED** [review]

File: helpers/theme.js (hunk 2 above)
Comment (odalys-dataport) [reply helpers/theme.js:38]: I wasn't sure if there ever is a difference between short_title and title so I figured if short_title is used in the headline, I might as well use short_title here too

**Pull request closed without merging**